src/ess/isissans/io.py

```diff
@@ -23,46 +21,6 @@
 CalibrationFilename = NewType('CalibrationFilename', str)
 
 
-def read_xml_detector_masking(filename: PixelMaskFilename) -> MaskedDetectorIDs:
```

Comment:
Moved to `ess/sans`: While strictly speaking ISIS-specific, our `ess.loki` unit tests were using this, introducing an annoying dependency of the `ess.loki` unit tests on `ess.isissans`.